### PR TITLE
feat(e2e): twenty-e2e-testing with playwright

### LINKED ISSUE #6360: Setup E2E testing
Twenty starts to have a lot of features and different use cases to test.

When doing refactors we usually try to test the app manually to make sure everything is working as expected.

But as it can now take a long time, we would like to have at least manual e2e tests to launch locally while developing or reviewing PR that involve refactors.

We already have integration tests with storybook and a lot of unit tests with jest, we also have visual regression testing with chromatic.

So e2e should focus on the most basics things to detect bugs : 
- Ensuring login works
- Ensuring there's no bug or errors when we visit each page
- Ensuring different workflows are executing without bugs or errors

Where e2e would compound efficiently is to offer us a minimal test coverage on a lot of small workflows because tests are easy to write.

We would see later how to integrate this in CI as it is not the main priority right now.

TODO : 
- [ ] Create a new package twenty-e2e-testing and install playwright following its documentation
- [ ] Setup environment variables for the base url, etc.
- [ ] Create a basic test that visits /objects/companies and that verifies there's a table shown

### PR DESCRIPTION
**Pull request by Nabhag8848**
## ISSUE (e2e) - Introduces e2e for twenty

 - Closes #6360
 
##  Description

- [x] Create Package. 
- [x] Setup environments such as baseUrl.
- [x] ignore CI configuration for now.
- [x] write a simple test to check if table is visible in companies tab.

### Running test with UI

```
yarn run test:e2e:ui
```
https://github.com/user-attachments/assets/a7b7ae35-8898-461e-8c7c-d3e4e9515aeb

### Running all test and seeing report

```
yarn run test:e2e
yarn run test:e2e:report
```

https://github.com/user-attachments/assets/2558a1f9-97cc-4f06-86f0-806f207eac5a

### CODE REVIEW

**Comment by Nabhag8848** [comment]
@lucasbordeau I have few more questions before we can review this. 

- What all major browsers we need to configure ?   
- what about mobile viewports ? if we need this rn, can you provide more details, will add. 
- Do we need to include this in documentation, it will be, can you point out how we are adding docs - example of it ?

File: packages/twenty-e2e-testing/.env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Note that provide always without trailing forward slash to have expected behaviour
+FRONTEND_BASE_URL="http://localhost:3001"
```
[the remark this answers is not in the document]
Comment (Nabhag8848) [reply packages/twenty-e2e-testing/.env.example:2]: Thanks doing lesss goo

**Comment by lucasbordeau** [comment]
@Nabhag8848 Thanks, nice work.

- Chrome is ok for now
- No need to cover mobile viewport for now, we'll see later
- We'll see for documentation later

**Comment by lucasbordeau** [comment]
You can let Firefox and Safari as they're already here.

**Comment by Nabhag8848** [comment]
@lucasbordeau made changes 🔥, lemme know if we need any more changes. also we can work next on #4832 or on adding CI config to same, whichever has P0 priority.

**Comment by lucasbordeau** [comment]
@Nabhag8848 Could you please check the problem with yarn and the CI ? It seems to be related to the packages you installed.

**Comment by Nabhag8848** [comment]
@lucasbordeau

**Review by lucasbordeau — APPROVED** [review]
Nice work thank you.

**Pull request merged**